plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDPreferencePage.java

```diff
@@ -128,11 +148,10 @@ private void createVisibilityGroup(DBPPreferenceStore store, Composite composite
         }
     }
 
-    private void createStyleGroup(DBPPreferenceStore store, Composite composite)
-    {
+    private void createStyleGroup(DBPPreferenceStore store, Composite composite){
         ERDViewStyle[] enabledStyles = ERDViewStyle.getDefaultStyles(store);
 
-        Group elemsGroup = UIUtils.createControlGroup(composite, ERDUIMessages.erd_preference_page_title_attribute_style, 1, GridData.VERTICAL_ALIGN_BEGINNING | GridData.FILL_HORIZONTAL, 0);
+        Group elemsGroup = UIUtils.createControlGroup(composite, ERDUIMessages.erd_preference_page_title_attribute_style, 1, GridData.FILL_BOTH, 0);
```

Comment:
Could you attach the new preference page screenshot?

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/32031248/b1d4fdba-0403-46d4-a519-6f4b087b59fd)